e2e/util/wait.go

```diff
@@ -77,15 +77,43 @@ func WaitForDRPCDelete(ctx types.Context, cluster types.Cluster, name, namespace
 }
 
 func WaitForNamespaceDelete(ctx types.Context, cluster types.Cluster, name string) error {
+	log := ctx.Logger()
 	obj := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
 		},
 	}
 
+	managed, err := isManagedByRamenE2e(ctx, cluster, obj)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			log.Debugf("Namespace %q not found in cluster %q, no need to wait.", name, cluster.Name)
+
+			return nil
+		}
+
+		return err
+	}
+
+	if !managed {
+		log.Debugf("Namespace %q in cluster %q is not managed by ramen-e2e, skipping wait for namespace deletion.",
+			name, cluster.Name)
+
+		return nil
+	}
```

Comment:
This change is not needed. We don't care if the namespace is managed or not when we wait.

Reply:
With delete and wait now separated, waiting for deletion of non-ramen-e2e namespaces (without labels) will timeout since we skip deleting them but still wait. Any alternative approaches to handle this?

Reply:
Right, in the current way we need to check the label twice and skip. We could skip the log but it may be helpful for debugging.